node/rpc-client/src/client.rs

```diff
@@ -176,11 +176,22 @@ where
 
 /// Filecoin RPC client interface methods
 pub mod filecoin_rpc {
+    use address::json::AddressJson;
     use jsonrpc_v2::Error;
+    use wallet::json::KeyInfoJson;
 
     use crate::call_params;
-    use rpc_api::{auth_api::*, chain_api::*};
+    use rpc_api::{
+        auth_api::*,
+        chain_api::*,
+        wallet_api::{
+            WalletBalanceParams, WalletExportParams, WalletHasParams, WalletImportParams,
+            WalletNewParams, WalletSetDefaultParams, WalletSignParams, WalletSignResult,
+            WalletVerifyParams,
```

Comment:
I recommend a `*` import here. We use all of them from this module.

Reply:
:+1: